crates/kitsune_p2p/proxy/src/tls_tests.rs

```diff
@@ -0,0 +1,73 @@
+use crate::*;
+use futures::stream::StreamExt;
+
+fn init_tracing() {
+    let _ = ghost_actor::dependencies::tracing::subscriber::set_global_default(
+        tracing_subscriber::FmtSubscriber::builder()
+            .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+            .finish(),
```

Comment:
observability is now it's own crate fyi, so you could use `observability::test_run()` but also not a big deal. Might be more important once we have open telemetry etc.

Reply:
I'm being pretty cautious about dependencies because I'm still hoping to extract kitsune to a different repo and don't want to run into circular mono-repo dependency issues... But, I suppose at that point we could extract observability to a third repo...